### PR TITLE
Fix browsers requirements

### PR DESCRIPTION
**Pull request by skjnldsv**
- Show a list of supported browsers
- Remove outdated talk screensharing warning

### CODE REVIEW

File: user_manual/webinterface.rst [hunk 1]
```diff
@@ -11,31 +11,29 @@ your Nextcloud server URL (e.g. cloud.example.com) and enter your username and p
...
-
-* Apple **Safari**
+* Apple **Safari** (Desktop and iOS)
 
 .. No need to translate
-
 * Microsoft **Edge**
 
+.. note:: Not all versions are supported. Nextcloud is tested and built to work with `these versions only.
+   <https://browserslist.dev/?q=PjAuMjUlLCBub3Qgb3BfbWluaSBhbGwsIG5vdCBkZWFkLCBGaXJlZm94IEVTUg==>`
```
Comment (nickvergessen) [inline user_manual/webinterface.rst:30]:
```suggestion
   <https://browserslist.dev/?q=PjAuMjUlLCBub3Qgb3BfbWluaSBhbGwsIG5vdCBkZWFkLCBGaXJlZm94IEVTUg==>`_
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: user_manual/webinterface.rst [hunk 2]
```diff
@@ -11,31 +11,29 @@ your Nextcloud server URL (e.g. cloud.example.com) and enter your username and p
...
-For the best experience with the Nextcloud web interface, we recommend that you use the 
-latest and supported version of a browser from this list:
-
+For the best experience with the Nextcloud web interface, we recommend that
+you use the latest and supported version of a browser from this list:
 
 .. No need to translate
-
-* Google **Chrome**/Chromium
+* Google **Chrome**/Chromium (Desktop and Android)
```
Comment (artonge) [inline user_manual/webinterface.rst:18]: Still no need to translate if we add 'and' ?
Same for others

Comment (nickvergessen) [reply user_manual/webinterface.rst:18]: https://www.transifex.com/nextcloud/nextcloud-user-documentation/translate/#de/$/184482352

Doesn't seem to work anyway

**Comment by nickvergessen** [comment]
Bildschirmfoto vom 2022-11-22 11-04-14

**Review by nickvergessen — APPROVED** [review]

**Review by artonge — APPROVED** [review]

**Pull request merged**